Tasks/HelmDeployV0/src/helmcommands/helmpackage.ts

```diff
@@ -26,6 +27,10 @@ export function addArguments(helmCli: helmcli) : void {
     }
 
     if(destination) {
+        if(!fs.existsSync(destination))
```

Comment:
instead of fs you can use if(!tl.exist(destination)) {.. }

Reply:
Okay. I'll make these changes.